AsyncDisplayKit/ASCollectionView.h

```diff
@@ -319,15 +319,26 @@ NS_ASSUME_NONNULL_BEGIN
  *
  * @param indexPath The index path of the requested node.
  *
- * @returns a node for display at this indexpath.  Must be thread-safe (can be called on the main thread or a background
- * queue) and should not implement reuse (it will be called once per row).  Unlike UICollectionView's version, this method
+ * @returns a node for display at this indexpath. This will be called on the main thread and should not implement reuse (it will be called once per row).  Unlike UICollectionView's version, this method
```

Comment:
This should still be on three lines - it is causing horizontal scrolling on Github :)  Though we don't have a fixed max line length, I think this is longer than some of the next-longest in this file.